bitnami-services/upgrading-tkr.hbs.md

```diff
@@ -0,0 +1,143 @@
+    ```console
+    kubectl get classclaim <CLASS-CLAIM-NAME> -n <CLASS-CLAIM-NAMESPACE> -ojsonpath="{.status.provisionedResourceRef}"
+    ```
+
+    For example, for a MongoDB claim the output will look like this:
+
+    ```json
+    {"apiVersion":"bitnami.database.tanzu.vmware.com/v1alpha1","kind":"XMongoDBInstance","name":"mongodb-zfjr5"}
+    ```
+
+1. Find the newest composition revision for your resource type:
+
+    ```console
+    kubectl get compositionrevisions
+    ```
+
+    In our example:
+
+    ```
+    NAME                                                             REVISION   XR-KIND               XR-APIVERSION                                 AGE
+    ...
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138      4          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h4m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-889eaeb      1          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h29m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-d869e8c      2          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h29m
+    xmongodbinstances.bitnami.database.tanzu.vmware.com-f1f3fe9      3          XMongoDBInstance      bitnami.database.tanzu.vmware.com/v1alpha1    3h5m
+    ...
+    ```
+
+    Take not of the name of the *highest* revision. In our example, this is revision 4 (`xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138`).
+
+1. Edit your managed resource:
+
+    ```console
+    kubectl edit <RESOURCE-API> <RESOURCE-NAME>
+    ```
+
+    In our example: 
+
+    ```console
+    kubectl edit xmongodbinstances.bitnami.database.tanzu.vmware.com mongodb-zfjr5
+    ```
+
+1. Change the resource `compositionRevisionRef` to point to the new composition revision. In our example:
+
+    ```yaml
+    apiVersion: bitnami.database.tanzu.vmware.com/v1alpha1
+    kind: XMongoDBInstance
+    metadata:
+      # ...
+    spec:
+      compositionRef:
+        name: xmongodbinstances.bitnami.database.tanzu.vmware.com
+      compositionRevisionRef:
+        name: xmongodbinstances.bitnami.database.tanzu.vmware.com-734d138
+    ```
+
+1. Check that the resource is ready again:
+
+    ```console
+    kubectl get <RESOURCE-API> <RESOURCE-NAME>
+    ```
+
+    In our example: 
+
+    ```console
+    kubectl get xmongodbinstances.bitnami.database.tanzu.vmware.com mongodb-zfjr5
+    ```
+
+    The output is:
+
+    ```console
+    NAME            SYNCED   READY   COMPOSITION                                           AGE
+    mongodb-zfjr5   True     True    xmongodbinstances.bitnami.database.tanzu.vmware.com   3h24m
+    ```
+
+## <a id="tap-16"></a>If staying on TAP 1.6 or older
+
+Services claimed in TAP 1.6 won't be able to start on TKR 1.26.
+We recommend a workaround based on mutating pods via a mutating webhook. For example, using
+[Kyverno](https://kyverno.io/), the following `ClusterPolicy` will apply the necessary modifications to every
```

Comment:
**question** Is there any way we could limit this to only bitnami services pods?

Reply:
I have added a label selector!